config/locales/client.en.yml

```diff
@@ -352,6 +352,7 @@ en:
             prompt: "This post contains non-captioned images. Would you like to enable automatic AI captions on image uploads? (This can be changed in your preferences later)"
             confirm: "Enable"
             cancel: "Don't ask again"
+          error: "There was an error generating a caption for the image."
```

Comment:
We should still surface the rate limit error. Better for users to know what happened, no? 

Also, does the switch to toasts mean that upload processing isn't blocked now?

Reply:
Yeah, if we now have toasts, we can show the actual "You have excedeed your AI caption limit. Wait 1 minute and try again" message.